frontend/src/views/HomePage/discover/DiscoverView.tsx

```diff
@@ -11,9 +13,12 @@ interface DiscoverProps {
     priceRangeMin: number;
     priceRangeMax: number;
     priceRangeDefault: number;
-    productsData: [] | undefined;
+    productsData: AuctionItem[] | [];
```

Comment:
the type `AuctionItem[] | []` is redundant, since an empty array is valid for `AuctionItem[]` type